### ISSUE TITLE
Attempted reactions of RA-0001351 with protected cysteine under buffer conditions aiming to confirm that the made cysteine-RA1351 adduct was formed under buffer conditions.

### ISSUE THREAD

**Issue by qxsml** [StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study#7]
A buffer was prepared according to the buffer that was used in Gel based assay for RA-0001351.
25 mM Tris-HCl (pH ~8) 
10 % DMSO 
50 mM NaCl 
0.2 mM EDTA 
1.25 mM MgCl2.

A few reactions were conducted as shown below:

![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/7a996650-dfa7-4c6e-8b71-11ed83e8993d)

However, RA-1351 was not consumed under these conditions.
![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/b403261e-7ffd-48fa-8c36-e2855ed495e6)

Then, another sets of reactions were carried out using more equiv. of cysteine. It turned out cysteine was oxidized to disulfide (Cystine). 

_**Only a trace signal of the adduct was observed.**_   

![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/aedf99b4-e07e-4aa8-963b-35e42e2e9d24)
Reaction 9 with added adduct:
![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/9af69a0e-467a-4ef9-8615-32ce3cc9d40c)
TCEP was used to prevent cysteine oxidation but the adduct signal couldn't be detected (as above).

Further optimizations were carried out including carry out the reaction in darkness under inert atmosphere, change the temperature, etc. But a decent UV signal of the adduct could not be observed.

**Comment by qxsml** [comment]
Using Ion-extraction to extract ion with MS 311 Da, found a MS peak at the same retention time as the trace signal somewhat proved this trace peak is the peak of the adduct. 
![Rxn 7 ion extraction with substitles](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/5c27a335-a33b-48c0-b077-f7636c8b916a)

**Comment by qxsml** [comment]
I thought the reason that I can't observe the desired product peak under buffer conditions is that cysteine was oxidized to cystine, but I finally figured out the issue is RA-0001351 was not fully dissolved under the buffer conditions. 

Previously, the solution turned cloudy upon adding RA-0001351 in DMSO to the buffer. However, I hoped that more 1351 would dissolve as the dissolved 1351 reacted.

Here, I'm showing two reactions and their LCMS analysis for comparison under identical conditions except the amount of buffer was used.
The solution of rxn 15 is cloudy while rxn 18 is clear.
![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/14cbc7a6-e69a-4cc5-81cf-0600dd1e6813)
![R15](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/a8110141-48db-4f5c-809f-f771c9bc9b2e)

![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/0eb4f90a-9b3d-4b82-b29d-a39aad8ba9f5)
![image](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/24120bf8-44d8-4e5e-b3b4-22ef007e23e7)
![R18 ms](https://github.com/StructuralGenomicsConsortium/CNP16-iminoquinone-SarsRdRp-mechanstic-study/assets/147162201/0832ec65-fdff-4bd8-94e6-1b945c57d40a)

I also tried to use 1 uL 50 mM and 1 uL 10 mM solution of RA-0001351. Neither cysteine adduct nor RA-1351 was detected under such conditions.
For rxn 18, 20 uL buffer is the maximum amount of buffer to use. I tried to use 30 uL but the solution turned cloudy.


An LCMS report of Rxn 18 is attached below.
[R18.pdf](https://github.com/user-attachments/files/15806289/R18.pdf)

**Comment by mattodd** [comment]
@qxsml so this means that the adduct *does* form under these conditions? Previously the NMR was suggesting that it formed and the LCMS was not?

**Comment by qxsml** [comment]
> @qxsml so this means that the adduct _does_ form under these conditions? Previously the NMR was suggesting that it formed and the LCMS was not?

Yes, it does form under these conditions. Previously the NMR experiment was conducted in DMSO-D6, and I observed an adduct formation. Then I synthesized the adduct in DMF. Finally, I tried this reaction using buffer as the solvent and try to observe the adduct Peak in LCMS but I couldn't find a decent peak, only a trace signal as indicated in this issue
